ios/admob_flutter.podspec

```diff
@@ -16,12 +16,8 @@ Admob plugin that shows banner ads using native platform views.
   s.source_files = 'Classes/**/*'
   s.public_header_files = 'Classes/**/*.h'
   s.dependency 'Flutter'
-  
   # https://firebase.google.com/docs/ios/setup
-  s.dependency 'Firebase/Analytics', '~> 6.30'
-  s.dependency 'Firebase/AdMob'
```

Comment:
Per ios [getting started](https://developers.google.com/admob/ios/quick-start) these are not required for ads. Further, [optional step 5](https://firebase.google.com/docs/admob/ios/quick-start#analytics-and-firebase) covers adding analytics for improved features.

Removing these resolves the versioning conflict and technically isn't required to use this package.